goal_src/jak1/engine/level/level-info.gc

```diff
@@ -1019,6 +1019,7 @@
       (new 'static 'continue-point
```

Comment:
@Bintendo42 any other skips using deload I'm forgetting? this should cover the all flies boggy deload past klaww too

Reply:
for at least useful ones i think thats it. im pretty sure there are technically other ones but theyre completely useless

Reply:
yea you could definitely do others but these were all the ones I knew were useful